src/components/Home.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-trailing-spaces */
```

Comment:
- [ ] Same here. I can see you have disabled the `eslint` most of the pages. Please try to fix them as they are included just to make your code standard.